genie-agent/src/main/java/com/netflix/genie/agent/cli/CommandsConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+import com.beust.jcommander.JCommander;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+class CommandsConfig {
+
+    @Bean
+    GlobalAgentArguments globalAgentArguments() {
+        return new GlobalAgentArguments();
+    }
+
+    @Bean
+    JCommander jCommander(
```

Comment:
I'm surprised spring is able to pick up all these package protected beans within a package protected class...